chrome/content/zotero/elements/contextPane.js

```diff
@@ -55,7 +55,7 @@
 			if (!(viewType in viewTypeMap)) {
 				throw new Error(`ContextPane.viewType must be one of ["item", "notes"], but got ${viewType}`);
 			}
-			this._panesDeck.setAttribute("selectedIndex", viewTypeMap[viewType]);
+			this._panesDeck.selectedIndex = viewTypeMap[viewType];
```

Comment:
This sort of thing should be in a separate commit (with the other instance), prefixed with "fx115: ", so we can document it as a necessary change and link to it as an example.

Reply:
OK, I'll split them

Reply:
Just splitted, but haven't renamed to fx115 prefixed. Will rename commits later